src/Tizen.NUI/src/public/BaseComponents/TextEditorBindableProperty.cs

```diff
@@ -935,5 +935,22 @@ public partial class TextEditor
             Tizen.NUI.Object.GetProperty((System.Runtime.InteropServices.HandleRef)textEditor.SwigCPtr, TextEditor.Property.MaxLength).Get(out temp);
             return temp;
         }));
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty FontSizeScaleProperty = BindableProperty.Create(nameof(FontSizeScale), typeof(float), typeof(TextEditor), default(float), propertyChanged: (BindableProperty.BindingPropertyChangedDelegate)((bindable, oldValue, newValue) =>
+        {
+            var textEditor = (TextEditor)bindable;
+            if (newValue != null)
+            {
+                Tizen.NUI.Object.SetProperty((System.Runtime.InteropServices.HandleRef)textEditor.SwigCPtr, TextEditor.Property.FontSizeScale, new Tizen.NUI.PropertyValue((float)newValue));
```

Comment:
According to rule [CA2000](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2000), disposable object need to be disposed before losing scope.
How about dispose a `PropertyValue` using `using statement`?
It is safe to dispose a created property after set it to the TextEditor.
```C#
using (var property = new Tizen.NUI.PropertyValue((float)newValue))
{
    Tizen.NUI.Object.SetProperty((System.Runtime.InteropServices.HandleRef)textEditor.SwigCPtr, TextEditor.Property.FontSizeScale, property);
}
```

Reply:
Thank you for suggestion. I'll fix it with the "using" statement.